tf-modules/s3-replicator/main.tf

```diff
@@ -5,12 +5,23 @@ provider "aws" {
 
 data "archive_file" "replicator_package" {
   type        = "zip"
-  source_file = "index.js"
-  output_path = "build/replicator.zip"
+  source_file = "${path.module}/index.js"
+  output_path = "${path.module}/build/replicator.zip"
+}
+
+resource "aws_security_group" "s3_replicator_lambda" {
```

Comment:
create our own security group

Reply:
updated this in https://github.com/nasa/cumulus/pull/1135/commits/c22f1cc557f8e02f23e4a5538396b6c814f3f7c9

`s3-replicator` will create it's own security groups only if they aren't passed in

Reply:
I've been having modules create their own security groups where needed. I can't think of a reason that a user of this module would want to manually create a SG that will be used by the lambda function.